kolibri/auth/backends.py

```diff
@@ -21,7 +21,7 @@ def authenticate(self, username=None, password=None, facility=None):
         :param facility: a Facility
         :return: A FacilityUser instance if successful, or None if authentication failed.
         """
-        users = FacilityUser.objects.filter(username=username)
+        users = FacilityUser.objects.filter(username__iexact=username)
```

Comment:
If we decide to "cast" all usernames to lowercase on the backend, we should also do that on the front-end as soon as it's entered in the input box.

Otherwise it could be frustrating to create SomeNewUser and have that unexpectedly show up as somenewuser later on.